@xen-orchestra/web/src/components/account-menu/AccountMenu.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <MenuList border :disabled placement="bottom-end">
+  <MenuList border :disabled placement="bottom-end" no-border>
```

Comment:
Since you inverse the logic you don't need `border` and `no-border` in this case
```suggestion
  <MenuList :disabled placement="bottom-end">
```